website/docs/progressive-delivery/progressive-delivery-flagger-install.mdx

```diff
@@ -9,88 +9,75 @@ import TierLabel from "../_components/TierLabel";
   {frontMatter.title} <TierLabel tiers="Enterprise" />
 </h1>
 
-[Flagger](https://docs.flagger.app/) is a progressive delivery operator for Kubernetes. It is
-designed to reduce risks when introducing new software versions and to improve time to delivery
-through automating production releases. Weave GitOps Enterprise's UI allows you to view the state of
-these progressive delivery rollouts, and how they are configured using Flagger's
-[canary](https://docs.flagger.app/usage/how-it-works#canary-resource) object, through the
-Applications > Delivery view.
+Built upon the core tenets of continuous integration and continuous delivery (CI/CD), [progressive delivery](https://www.weave.works/blog/progressive-delivery-checklist) involves gradually rolling out features to small groups of select users to balance performance with speed. Developers and DevOps teams use fine-grained controls to minimize the risks of pushing new features to the production environment. If the newly released feature proves to be stable and performant, it can then be released to all users.
+
+[Flagger](https://docs.flagger.app/) is a progressive delivery operator for Kubernetes and part of the Flux family of open source projects. It reduces the risk of introducing new software versions and automates production releases to improve your time to delivery. Flagger implements deployment strategies—canary releases, A/B testing, Blue/Green mirroring—using a service mesh (App Mesh, Istio, Linkerd, Kuma, Open Service Mesh) or an ingress controller (Contour, Gloo, NGINX, Skipper, Traefik, APISIX) for traffic routing. For release analysis, Flagger can query Prometheus, InfluxDB, Datadog, New Relic, CloudWatch, Stackdriver, or Graphite. For alerting it uses Slack, MS Teams, Discord, and Rocket. Using Flux allows us to manage our cluster applications in a declarative way through changes in a Git repository.
```

Comment:
What does "strategies-canary" mean? And "mirroring-using"?

Reply:
Those should be showing up as em dashes—are they not?

Reply:
Not in the diff! If these are rendered out to em dashes, then :+1: (GitHub doesn't show a rendered view, probably because it doesn't know the file extension.)

Reply:
Oh weird